physicsnemo/models/dlwp_healpix_layers/healpix_layers.py

```diff
@@ -190,7 +189,12 @@ class HEALPixPadding(th.nn.Module):
     Orientation and arrangement of the HEALPix faces are outlined above.
     """
 
-    def __init__(self, padding: int, enable_nhwc: bool = False):
+    def __init__(
+        self,
+        padding: int,
+        enable_nhwc: bool = False,
+        hpx_padding_mode: str = "karlbauer",
```

Comment:
Not strictly needed for this PR, but it would be great if this module can be compatible with `torch.compile`. Either in this PR or later, would be good to make sure this compiles without errors and benchmark to see how much of the perf gap to the CUDA version can be made up.